influxdb_iox/src/commands/run/ingester.rs

```diff
@@ -94,6 +95,9 @@ pub async fn command(config: Config) -> Result<()> {
         );
     }
 
+    // Ensure panics are fatal when running in this server mode.
```

Comment:
I think this comment would be more helpful if it encoded the reason *why* the process is in "abort on panic"  mode which is not necessarily obvious by looking at the code.

Perhaps something like this:

```suggestion
    // Ensure panics are fatal when running in this server mode.
    // Often when the ingester panics there is no safe way to recover, 
    // so force a process restart. 
```